tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -349,6 +349,28 @@ public ExecutorService getThreadPoolForDeleteRange() {
     return res;
   }
 
+  public ExecutorService getThreadPoolForIsAlive() {
+    if (isAliveThreadPool == null) {
+      synchronized (this) {
+        if (isAliveThreadPool == null) {
+          isAliveThreadPool =
+              new ThreadPoolExecutor(
+                  20,
```

Comment:
Why use 20?
I think it should be associated with core numbers.
Runtime.getRuntime().availableProcessors();